webhooks-extension/README.md

```diff
@@ -50,6 +56,10 @@ curl -d "${data}" -H "Content-Type: application/json" -X POST http://localhost:9
 
 Reference the [Knative eventing GitHub source sample](https://knative.dev/docs/eventing/samples/github-source/) to properly create the `accesstoken` secret. This is the secret that is used to create GitHub webhooks.
 
+If using Helm, you can also specify a Helm release name that enables you to do `helm install --upgrade` as expected, otherwise our sink will include the short Git commit ID in your Helm release name.
+
+Provide `releasename` (which must be less than 64 characters in length) in the POST request. This is an optional field.
```

Comment:
Since this sentence refers to the API definition, please move this sentence to the extension's api definition README: https://github.com/tektoncd/experimental/blob/master/webhooks-extension/cmd/extension/README.md#post-endpoints.